modules/shortcodes/vimeo.php

```diff
@@ -96,7 +96,7 @@ function vimeo_shortcode( $atts ) {
 	 */
 	$height = (int) apply_filters( 'vimeo_height', $height );
 
-	$url = esc_url( set_url_scheme( "http://player.vimeo.com/video/$id" ) );
+	$url = esc_url( set_url_scheme( "https://player.vimeo.com/video/$id" ) );
```

Comment:
the `set_url_scheme` will return `http` or `https` depending on result of `is_ssl`. If we want to always use `https` (any real downsides anyone? I'm okay with that vs protocol neutral), we can omit `set_url_scheme`.